### DISCUSSION TITLE
Map generators should list and allow input of seeds

### DISCUSSION THREAD

**Discussion by janisozaur** [OpenRCT2/OpenRCT2#15767] [Feature requests]
**Commit/Build:** 4e3a38b 

Map generators allow generating map based on PRNG. This PRNG should be verified to be the same across platforms and UI should be extended to:
1. List past/current seeds (I imagine this could be an actual list with 5 or so latest used entries, so a user can easily switch between them)
2. Allow entering a custom seed for a map

**Comment by IntelOrca** [comment]
Alterations to the map generator - just even an extra call to the PRNG in the middle of the generation algorithm could change the map a seed produces dramatically between versions of the game. How do you suggest we solve this?

**Comment by janisozaur** [comment]
I don't think that's worth addressing. The use case I imagine for this is when you are in the game and want to generate some map. You would open the terrain generator and on each click you get a new map without any way of reverting to what you saw.
